src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx

```diff
@@ -59,7 +59,66 @@ export interface IRepeatingGroupTableProps {
 
 const theme = createTheme(altinnAppTheme);
 
+// Page padding could be moved to css-variables
+const ps = 24;
+const pm = 84;
+const pl = 96;
+const m = 15;
```

Comment:
I see the reasoning behind keeping these variable names short, but this way they don't make any sense for me. Could you add a comment to describe them at, least? :thinking: 

My interpretation:
- `ps` = padding, small
- `pm` = padding medium
- `pl` = padding large
- `m` = something else that is medium? :shrug:

Reply:
Also, is this kind-of a re-implementation of `FullWidthWrapper`? Could something from that component be re-used here?

Reply:
Never heard of that, will look into it

Reply:
Yes I guess it kind of is, but it also has styling to line up the cell-content with the page, so there are not that many lines to save by reusing this

Reply:
Sure, I could use the fullwidthwrapper with a small modification to it so i could add classnames through props